.github/workflows/use-latest-docker-image-for-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Use latest docker image for CI
```

Comment:
Maybe the name (and the filename can be):
`Force update stable Fluent image version`

Reply:
Done. Thank you.